target/linux/ramips/patches-5.10/810-uvc-add-iPassion-iP2970-support.patch

```diff
@@ -225,18 +225,16 @@ Signed-off-by: John Crispin <blogic@openwrt.org>
  	for_each_uvc_urb(uvc_urb, stream) {
 --- a/drivers/media/usb/uvc/uvcvideo.h
 +++ b/drivers/media/usb/uvc/uvcvideo.h
-@@ -203,7 +203,9 @@
- #define UVC_QUIRK_RESTORE_CTRLS_ON_INIT	0x00000400
+@@ -204,6 +204,8 @@
  #define UVC_QUIRK_FORCE_Y8		0x00000800
  #define UVC_QUIRK_FORCE_BPP		0x00001000
--
+ #define UVC_QUIRK_WAKE_AUTOSUSPEND	0x00002000
 +#define UVC_QUIRK_MOTION		0x00001000
 +#define UVC_QUIRK_SINGLE_ISO		0x00002000
```

Comment:
Please use new numbers for these new defines.

Reply:
Done will push soon